app/Http/Controllers/Backend/WikiController.php

```diff
@@ -10,19 +11,17 @@ class WikiController extends Controller {
     private static $with = ['wikiDb', 'wikiQueryserviceNamespace', 'settings'];
```

Comment:
I don't think this is a magic Laravel feature, so now that it isn't being used on line 22, it can be removed?

```suggestion
```

Reply:
yep! thanks, I forgot to remove the line